### PR TITLE
E1fabre miami

### PR DESCRIPTION
**Pull request by e1fabre**
@struther  check out my first pull request

closes issue#8

### CODE REVIEW

**Review by mohammadkalash — COMMENTED** [review]
I like those suggestions.  Any of these offer goat cheese appetizers?

File: e1fabre-miami.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+### Where to Eat
+
+-Sponge Docks
```
Comment (struther) [inline e1fabre-miami.md]: @e1fabre 
Is there a sponge docks in Miami?

File: e1fabre-miami.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+-Dunein Brewery:beers:
+-Casa Tina's
```
Comment (struther) [inline e1fabre-miami.md]: @e1fabre 
looking for something near the U!

**Review by struther — COMMENTED** [review]
@e1fabre 
answer my questions, please
:rage4:

**Pull request merged**